packages/apps-config/src/api/chain/cere.ts

```diff
@@ -4,30 +4,24 @@
 // structs need to be in order
 /* eslint-disable sort-keys */
 
-// For the network after ERC20 integrated
 export default {
-  ChainId: "u8",
-  DepositNonce: "u64",
-  ResourceId: "[u8; 32]",
+  ChainId: 'u8',
```

Comment:
Have you tested this config on Testnet DEV and Testnet PROD?

Reply:
I tested this config on Testnet PROD. As it turned out, it fixes issues on PROD, but causes the same issues on DEV. We need to investigate this problem more deeply.

Reply:
This config works great with the latest node version on DEV2 testnet, so it is going to be merged.